test/lib/jdk/test/lib/jfr/EventNames.java

```diff
@@ -197,7 +197,7 @@ public class EventNames {
     public static final String ResidentSetSize = PREFIX + "ResidentSetSize";
 
     // JDK
-    public static final String FileForce  = PREFIX + "FileForce";
```

Comment:
Looks like just a whitespace change. The change is correct, but we avoid making whitespace only changes to files that don't otherwise have any other changes. I'd back this out.